Jonesy22/Perro
# Per 33 user session

Sessions are working for a user logged into Google!

Test plan:

- Start up frontend: `npm start`
- Start up backend: `nodemon express-backend/index.js`
- Login to oregon state VPN
- Confirm session ID appears in the backend terminal and no console errors on the browser.
- Try logging in and out.
